src/main/java/org/mockito/internal/junit/JUnitRule.java

```diff
@@ -70,11 +59,14 @@ private Throwable evaluateSafely(Statement base) {
     }
 
     public MockitoRule silent() {
-        return new JUnitRule(logger, Strictness.LENIENT);
+        return strictness(Strictness.LENIENT);
     }
 
     public MockitoRule strictness(Strictness strictness) {
-        this.listener.setStrictness(strictness);
+        this.strictness = strictness;
+        if (session != null) {
```

Comment:
Can you write a comment explaining when session is null? Useful for folks who don't know this code very well and future maintainers!

Reply:
Yep, will do!